lib/ohai/plugins/windows/virtualization.rb

```diff
@@ -44,6 +44,14 @@
       virtualization[:system] = "kvm"
       virtualization[:role] = "guest"
       virtualization[:systems][:kvm] = "guest"
+    when "American Megatrends Inc."
```

Comment:
In reviewing a few different guests running on different version of hyper-v, the smbiosbiosversion property appears to be '090006' consistently.  Reasonable to evaluate that when the manufacturer is 'American Megatrends Inc.'?